src/dmd/root/outbuffer.d

```diff
@@ -448,12 +448,12 @@ struct OutBuffer
 /****** copied from core.internal.string *************/
 
 private:
-pure:
-nothrow:
-@nogc:
 
 alias UnsignedStringBuf = char[20];
 
+pure
+nothrow
+@nogc
```

Comment:
Please put them on the function declaration directly instead of this weird syntax